[2025-10-21T18:00:50.582+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-10-21T18:00:50.635+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: airbnb_etl_pipeline.transform_airbnb_forsale manual__2025-10-21T17:45:45.179461+00:00 [queued]>
[2025-10-21T18:00:50.650+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: airbnb_etl_pipeline.transform_airbnb_forsale manual__2025-10-21T17:45:45.179461+00:00 [queued]>
[2025-10-21T18:00:50.653+0000] {taskinstance.py:2306} INFO - Starting attempt 2 of 2
[2025-10-21T18:00:50.671+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): transform_airbnb_forsale> on 2025-10-21 17:45:45.179461+00:00
[2025-10-21T18:00:50.682+0000] {standard_task_runner.py:63} INFO - Started process 403 to run task
[2025-10-21T18:00:50.689+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'airbnb_etl_pipeline', 'transform_airbnb_forsale', 'manual__2025-10-21T17:45:45.179461+00:00', '--job-id', '19', '--raw', '--subdir', 'DAGS_FOLDER/airbnb_etl_dag.py', '--cfg-path', '/tmp/tmpzqekqfwy']
[2025-10-21T18:00:50.695+0000] {standard_task_runner.py:91} INFO - Job 19: Subtask transform_airbnb_forsale
[2025-10-21T18:00:50.797+0000] {task_command.py:426} INFO - Running <TaskInstance: airbnb_etl_pipeline.transform_airbnb_forsale manual__2025-10-21T17:45:45.179461+00:00 [running]> on host 7b5c622c061b
[2025-10-21T18:00:50.900+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='airbnb_etl_pipeline' AIRFLOW_CTX_TASK_ID='transform_airbnb_forsale' AIRFLOW_CTX_EXECUTION_DATE='2025-10-21T17:45:45.179461+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-10-21T17:45:45.179461+00:00'
[2025-10-21T18:00:50.901+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-10-21T18:00:50.916+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-10-21T18:00:50.918+0000] {taskinstance.py:2905} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/models/taskinstance.py", line 465, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/models/taskinstance.py", line 432, in _execute_callable
    return execute_callable(context=context, **execute_callable_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/models/baseoperator.py", line 401, in wrapper
    return func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/operators/python.py", line 235, in execute
    return_value = self.execute_callable()
                   ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/operators/python.py", line 252, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/airflow/scripts/etl_script.py", line 1215, in transform2
    airdna_data_for_sale = pd.json_normalize(all_properties)
                                             ^^^^^^^^^^^^^^
NameError: name 'all_properties' is not defined
[2025-10-21T18:00:50.947+0000] {taskinstance.py:1206} INFO - Marking task as FAILED. dag_id=airbnb_etl_pipeline, task_id=transform_airbnb_forsale, run_id=manual__2025-10-21T17:45:45.179461+00:00, execution_date=20251021T174545, start_date=20251021T180050, end_date=20251021T180050
[2025-10-21T18:00:50.962+0000] {standard_task_runner.py:110} ERROR - Failed to execute job 19 for task transform_airbnb_forsale (name 'all_properties' is not defined; 403)
[2025-10-21T18:00:50.983+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 1
[2025-10-21T18:00:51.007+0000] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-10-21T18:00:51.011+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
